drivers/SmartThings/zigbee-motion-sensor/src/ikea/init.lua

```diff
@@ -63,6 +64,8 @@ local function zdo_binding_table_handler(driver, device, zb_rx)
       return
     end
   end
+  driver:add_hub_to_zigbee_group(0x0000) -- fallback if no binding table entries found
+  device:send(Groups.commands.AddGroup(device, 0x0000, "00"))
```

Comment:
@tpmanley this is how I parsed that command from the DTH, but would love some confirmation.

Reply:
@cjswedes @varzac , double check me here, but I think for a `CharString` the length is handled automatically so we can just pass an empty string here.

Reply:
Correct, just the value needs to be included

Reply:
oh, that `00` in the DTH was just encoding the length of the (empty) group name? Got it.